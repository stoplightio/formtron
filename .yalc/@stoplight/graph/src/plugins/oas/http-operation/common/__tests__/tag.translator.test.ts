import { translateToTags } from '../tag.translator';

describe('translateToTags', () => {
  test('should translate array of strings to tags', () => {
    expect(translateToTags(['a', 'b', 'c'])).toMatchSnapshot();
  });
});
