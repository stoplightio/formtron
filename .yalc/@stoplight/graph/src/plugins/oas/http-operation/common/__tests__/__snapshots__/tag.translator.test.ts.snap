// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`translateToTags should translate array of strings to tags 1`] = `
Array [
  Object {
    "name": "a",
  },
  Object {
    "name": "b",
  },
  Object {
    "name": "c",
  },
]
`;
